kolibri/utils/cli.py

```diff
@@ -245,6 +246,18 @@ def plugin(plugin_name, **args):
     conf.save()
 
 
+def set_default_language(lang):
+    """
+    Set the default language for this installation of Kolibri. Any running
+    instance of Kolibri needs to be restarted in order for this change to work.
+    """
+
+    from kolibri.utils import conf
+
+    conf.config['LANGUAGE_CODE'] = lang
```

Comment:
Maybe we should check the 'set' code against the `LANGUAGES` setting here to make sure that they don't set non-permitted languages by accident?